pkg/schedule/config/config_provider.go

```diff
@@ -62,6 +62,7 @@ type SchedulerConfigProvider interface {
 	GetHotRegionCacheHitsThreshold() int
 	GetMaxMovableHotPeerSize() int64
 	IsTraceRegionFlow() bool
+	GetPatrolRegionConcurrency() int
```

Comment:
Why not put the `GetPatrolRegionConcurrency` function in `CheckerConfigProvider` like `GetPatrolRegionInterval`?